portal/server/controllers/utilisation-report-service/utilisation-report-upload/utilisation-report-status.js

```diff
@@ -20,7 +20,7 @@ const getBankHolidays = async (userToken) => {
 const getReportPeriod = () => {
   const lastMonth = subMonths(new Date(), 1);
   const reportPeriod = format(lastMonth, 'MMMM yyyy');
-  const month = getMonth(lastMonth);
+  const month = getMonth(lastMonth) + 1; // Account for javascript 0-index for months
```

Comment:
I guess we're going to have to do this a lot because javascript. Maybe it'd be worth creating a shared `getOneIndexMonth` function that we can use in all these cases. Then we also wouldn't need to add a comment to explain the reason.

Reply:
Yep, I've added it :)